README.md

```diff
@@ -5,25 +5,50 @@ Create SaaS Boilerplate
 [![Downloads/week](https://img.shields.io/npm/dw/create-saas-boilerplate.svg?style=for-the-badge)](https://npmjs.org/package/create-saas-boilerplate)
 [![License](https://img.shields.io/npm/l/create-saas-boilerplate.svg?style=for-the-badge)](https://github.com/apptension/create-saas-boilerplate/blob/master/package.json)
 
+Create SaaS Boilerplate is a CLI tool designed to quickly set up a new local instance of the SaaS Boilerplate. It is a specialized tool that allows users to get started with the SaaS Boilerplate as quickly as possible. 
+
+
+---
+
+📖 **SaaS Boilerplate Documentation**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+
+🌟 **SaaS Boilerplate Page**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+
```

Comment:
Please add here the main usage which will be to use one of the following commands:
`yarn create saas-boilerplate`
`pnpm create saas-boilerplate`
`npn init saas-boilerplate`